README.md

```diff
@@ -387,11 +387,56 @@ Pathfinder has a monitoring API which can be enabled with the `--monitor-address
 
 ### Metrics
 
-`/metrics` provides a [Prometheus](https://prometheus.io/) metrics scrape endpoint. Currently only one type of metric is available:
-- `rpc_method_calls_total` counter, where the label key `method` should be used to point to a particular RPC method to retrieve that method's total call count, for example: `rpc_method_calls_total{method="starknet_getStateUpdate"}`.
-- `extpy_processes_launched_total` counter incremented each time python subprocess is launched
-- `extpy_processes_exited_total` counter with labels, incremented each time python subprocess exits normally
-- `extpy_processes_failed_total` counter, incremented each time python subprocess exits abnormally
+`/metrics` provides a [Prometheus](https://prometheus.io/) metrics scrape endpoint. Currently the following metrics are available:
+
+#### RPC related counters
+
+- `rpc_method_calls_total`,
+- `rpc_method_calls_failed_total`,
+
+You __must__ use the label key `method` to retrieve a counter for a particular RPC method, for example:
+```
+rpc_method_calls_total{method="starknet_getStateUpdate"}
+rpc_method_calls_failed_total{method="starknet_chainId"}
+```
+
+#### Python subprocess related counters
+
+- `extpy_processes_launched_total` incremented each time python subprocess is launched
+- `extpy_processes_exited_total` with labels, incremented each time python subprocess exits normally
+- `extpy_processes_failed_total` incremented each time python subprocess exits abnormally
+
+#### Sequencer related counters
+
+- `sequencer_requests_total`
+- `sequencer_requests_failed_total`
```

Comment:
As with the previous comment - I think we should all look at the naming scheme for all metrics because I could be biased.

`gateway` is more natural than `sequencer`. I'm just wondering now if we should keep both RO and W   requests under the same prefix, or follow the same distinction as the StarkWare api does, ie.:
- `feeder_gateway` is RO,
- `gateway` is W, which would now be: `call` and `add_transaction`.